etc/tones.tsv

```diff
@@ -1,9 +1,9 @@
-Category	Raw_ToneClass	Chao_Letters	Tone	Language_ID	Remark
-⁵	level	˧	³³	WhiteHmong	
-¹.³.⁴.⁷	s	 ˩	¹¹	WhiteHmong	
-³	d	˩˧	¹³	WhiteHmong	
-³	v	˧˦	³⁴	WhiteHmong	
-².⁴.⁶	g	˧˩	³¹	WhiteHmong	
-²	j	˥˧	⁵³	WhiteHmong	
-¹	b	˥	⁵⁵	WhiteHmong	
-⁴	m	˩'	²¹³	WhiteHmong	Heimbach stated that m and d might be the same. But I assign different tonal markings and tones to distinguish m and d.
\ No newline at end of file
+Language_ID	Name	Category	Transformed_Category	Raw_ToneClass	Chao_Letters	Tone	Remark

```

Comment:
We never talked about "Transformed_Category". Instead, I said: "CategoryInSource", and `Category`.

Reply:
changed